configs/sagan/README.md

```diff
@@ -0,0 +1,59 @@
+| sagan_128 | ImageNet |     w/o      |   1    |                       64x4x1                        |   1000000    | [ 31.5938 (980000) ](https://download.openmmlab.com/mmgen/sagan/sagan_imagenet1k_128_Glr1e-4_Dlr4e-4_ndisc1_b32x4_woReLUinplace_is-iter980000_20210730_163140-cfbebfc6.pth) | [ 34.7838 (950000) ](https://download.openmmlab.com/mmgen/sagan/sagan_imagenet1k_128_Glr1e-4_Dlr4e-4_ndisc1_b32x4_woReLUinplace_fid-iter950000_20210730_163431-d7916963.pth) | [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/sagan/sagan_128_woReLUinplace_imagenet1k_b64x4_Glr-1e-4_Dlr-4e-4_ndisc1.py) | [Log](https://download.openmmlab.com/mmgen/sagan/sagan_imagenet1k_128_Glr1e-4_Dlr4e-4_ndisc1_b32x4_woReLUinplace_20210730_163431_fid-d7916963_is-cfbebfc6.json) |
+
+'\*' Iteration counting rule in our implementation is different from others. If you want to align with other codebases, you can use the following conversion formula:
+```
+Total_iters* = Total_iter / (n_disc * n_grad_accu)
+```
+
+We also provide converted pretrain model from [Pytorch-StudioGAN](https://github.com/POSTECH-CVLab/PyTorch-StudioGAN).
+To be noted that, in Pytorch Studio GAN, **inplace ReLU** is used in generator and discriminator.
+
+|  Models   | Dataset  | Inplace ReLU | n_disc | n_grad_accu | Total Iters |  IS\*  |  FID\*  | IS\*\* | FID\*\* |                                                          Download                                                           |                                Original Download link                                |
+| :-------: | :------: | :----------: | :----: | :---------: | :---------: | :----: | :-----: | :----: | :-----: | :-------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------: |
+| sagan_32  | CIFAR10  |      w       |   5    |      1      |   100000    | 9.116  | 10.2011 | 8.680  | 14.009  |   [Download](https://download.openmmlab.com/mmgen/sagan/sagan_32_cifar10_convert-studio-rgb_20210730_153321-080da7e2.pth)   | [Download](https://drive.google.com/drive/folders/1FA8hcz4MB8-hgTwLuDA0ZUfr8slud5P_) |
+| sagan_128 | ImageNet |      w       |   1    |      1      |   1000000   | 27.367 | 40.1162 | 29.848 | 34.726  | [Download](https://download.openmmlab.com/mmgen/sagan/sagan_128_imagenet1k_convert-studio-rgb_20210730_153357-eddb0d1d.pth) | [Download](https://drive.google.com/drive/folders/1ZYaqeeumDgxOPDhRR5QLeLFIpgBJ9S6B) |
+
+
+* '\*\*' denote results evaluated with our pipeline.
```

Comment:
`**` and `***` are very ugly. You can just use names with or without `*` to indicate the difference.